xds/src/test/java/io/grpc/xds/ClientXdsClientDataTest.java

```diff
@@ -466,11 +474,11 @@ public void parseFaultAbort_withHeaderAbort() {
             .setPercentage(FractionalPercent.newBuilder()
                 .setNumerator(20).setDenominator(DenominatorType.HUNDRED))
             .setHeaderAbort(HeaderAbort.getDefaultInstance()).build();
```

Comment:
So these tests should be moved to `FaultFilter`.

Reply:
Done.